packages/polyfills-loader/src/createPolyfillsLoader.ts

```diff
@@ -29,9 +29,11 @@ const loadScriptFunction = `
       }
       script.src = src;
       script.onload = onLoaded;
-      attributes.forEach(att => {
-        script.setAttribute(att.name, att.value);
-      });
+      if(attributes){
+        attributes.forEach(att => {
```

Comment:
This needs to be changed to ES5 as well:

```suggestion
        attributes.forEach(function (att) {
```

Reply:
ups.. I've forgotten to copy this change. Totally agree about that